sourcegraph/sg.nvim
# cody: initial implementation

To test this:

You need to have `SRC_ACCESS_TOKEN` and `SRC_ENDPOINT` in your environment set up (so... you can do cloud [sourcegraph.com] and you get like 1000 completions in a day? something like that maybe).

Then you should have the `Cody*` commands available to you (you'll probably have to accept TOS of course, since we're submitting code places).

TODO (probably not in this PR):
- We should add some documentation generation, so we can document the different commands & functions we have.
